http-gateway/swagger.yaml

```diff
@@ -442,6 +442,24 @@ paths:
           $ref: '#/components/responses/badRequest'
         401:
           $ref: '#/components/responses/unauthorized'
+    delete:
+      tags:
+        - 'Pending Commands'
+      summary: 'Cancel updates metadata for a device'
+      description: |
+        Cancel all metadata updates for a single device. To cancel one pending update set 'correlationIdFilter' query parameter of pending command.
```

Comment:
Pending metadata update is identified by the `correlationId`. If not specified, all pending metadata updates will be canceled.